### PR TITLE
Username added at the top right corner of doccano

### PR DESCRIPTION
**Pull request by tusharmakkar08**
Fixes https://github.com/doccano/doccano/issues/604

Sample screenshots:
Unlogged in:
Screenshot 2020-04-02 at 6 33 46 PM

For admin:
Screenshot 2020-04-02 at 6 33 58 PM

For non admin:
Screenshot 2020-04-02 at 6 34 21 PM

For signout:
Screenshot 2020-04-02 at 6 34 06 PM

Also updated docker dev script because without debug cors issue was happening.

### CODE REVIEW

**Comment by tusharmakkar08** [comment]
Hey @Hironsan 

Review this PR and let me know the feedback.

Thanks.

File: frontend/components/organisms/layout/TheHeader.vue [hunk 1]
```diff
@@ -102,7 +99,8 @@ export default {
         { title: 'Sentiment Analysis', link: 'sentiment-analysis' },
         { title: 'Translation', link: 'translation' },
         { title: 'Text to SQL', link: 'text-to-sql' }
-      ]
+      ],
+      username: localStorage.getItem('name')
```
Comment (Hironsan) [inline frontend/components/organisms/layout/TheHeader.vue:103]: Use vuex instead of directly calling localStorage.

**Comment by Hironsan** [comment]
- Keep `mdi-dots-vertical`
- Don't show the user name in the header. If the name is long, it's ugly.
- Instead, show the name in the menu as follows(from YouTube):
image

File: frontend/components/organisms/layout/TheHeader.vue (hunk 1 above)
Comment (tusharmakkar08) [reply frontend/components/organisms/layout/TheHeader.vue]: Done.

**Comment by tusharmakkar08** [comment]
Thanks for the comments @Hironsan 

I have updated the PR as suggested. PFB the latest images: 

Screenshot 2020-04-03 at 11 57 45 AM

Screenshot 2020-04-03 at 11 57 36 AM

**Comment by Hironsan** [comment]
Thanks!

**Comment by tusharmakkar08** [comment]
Hey @Hironsan 

Since we are using vuex the username disappears when we do refresh. If we had implemented it via local storage it wouldn't have been the case. Please suggest. 

Thanks.

**Comment by Hironsan** [comment]
Oh, you are right.

Like auth token, we need to store user name into localStorage in Vuex:
https://github.com/doccano/doccano/blob/1f69b780a5f3ae24858788024c1b5f0a5ad99d28/frontend/store/auth.js#L37

**Comment by tusharmakkar08** [comment]
Okay.. I will take this task as part of next PR.

**Comment by kuraga** [comment]
@tusharmakkar08 , thanks very much!

**Pull request merged**